File: 이솔/프로그래머스/Level1/과일장수.py
def solution(k, m, score):
    answer = 0
    score.sort()
    a=len(score)
    b=a//m
    
    if b==0:
        answer=0
    elif b==1:
        answer=score[-m]
    else:
        for i in range(0,b):
            k=-i*m-m
            answer+=score[k]
    answer*=m
    return answer

File: 문민우/프로그래머스/level1/과일장수.java
package level2;

import java.util.Arrays;
import java.util.Collections;

public class 과일장수 {
    public int solution(int k, int m, int[] score) {

        int answer = 0;

        if(score.length < m)
            return answer;

        Integer[] scores = Arrays.stream(score).boxed().toArray(Integer[]::new);
        Arrays.sort(scores, Collections.reverseOrder());

        for(int i = 0; i <= scores.length - m; i += m) {
            answer += scores[i + (m - 1)] * m;
        }

        return answer;
    }
}
